### DISCUSSION TITLE
HELP WANTED [RESOLVED]

### DISCUSSION THREAD

**Discussion by asbjornenge** [asbjornenge/Docker.tmbundle#36] [Announcements]
Hi there!

Looking for maintainer for this repo. I have not worked on it for years and do not know what is what anymore 😅

Ping me if interested 👋

**Comment by princemaple** [comment]
Hey, I just cleaned up the issues and updated the new dockerfile with bash syntax. We should be up to speed now. I've turned on watch for this repo, so I'll get notifications for this repo in the future.

**Reply by asbjornenge** [reply]
Thanks a bunch mate ❤️🙌 Happy holidays 🎅🎄
